pkg/applicationserver/io/mqtt/format_json.go

```diff
@@ -16,7 +16,9 @@ package mqtt
 
 import (
 	"go.thethings.network/lorawan-stack/pkg/applicationserver/io/formatters"
-	"go.thethings.network/lorawan-stack/pkg/applicationserver/io/mqtt/topics"
+	mqtt "go.thethings.network/lorawan-stack/pkg/applicationserver/io/mqtt/topics"
```

Comment:
What is this alias for?

Reply:
This alias ensures that the JSON format for MQTT uses the MQTT topic format (and as such uses the `AcceptTopic ` that checks for `#` and automatically prepends the application name to it instead of the no-op one in `pubsub/layout`. I agree that the name of the alias perhaps should have been `mqtt_topics` or something akin to this.